airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceAcceptanceTest.java

```diff
@@ -1273,4 +1254,47 @@ protected AirbyteMessage createStateMessage(final DbStreamState dbStreamState, f
     }
   }
 
+  /**
+   * Method to add state_type: "standard" to state message for CTID based syncs
+   *
+   * @param syncState
+   */
+  protected void addStandardStateTypeToSyncState(final JsonNode syncState) {
```

Comment:
Do we need this? I thought with the new State manager we added, all state should have standard added to it. So no need to modify state messages here